### PR TITLE
fix(commerce): bugfix missing recs request configuration

### PR DESCRIPTION
**Pull request by BenKandelaars**
[CAPI-724](https://coveord.atlassian.net/jira/software/c/projects/CAPI/boards/1103?assignee=6168b023c5388b006911910a&selectedIssue=CAPI-724)

The real bug wasn't the missing configuration it was forgetting to await a promise of config. Recommendations controller now works as expected.

[CAPI-724]: https://coveord.atlassian.net/browse/CAPI-724?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Spuffynism — APPROVED** [review]
lgtm!

**Comment by olamothe** [comment]
Fix is good; however I think this points to a general problem in the project identified here: https://github.com/coveo/ui-kit/pull/3772 by Fred

TLDR: 🚢 , and we'll be taking care of improving the situation around this. TypeScript compiler should have caught the problem in the first place

**Review by olamothe — APPROVED** [review]
